scripts.py

```diff
@@ -601,6 +602,30 @@ def process_library(self, library):
         raise NotImplementedError()
 
 
+class LaneSweeperScript(LibraryInputScript):
```

Comment:
Most of this code is brought in from circulation, from the parent class for the scripts that cache OPDS feeds for each lane. My original intention was to have a separate script that updated the counts for each lane. But I think it's better to update the counts only when the materialized views are updated, since that's the only time the counts are likely to change significantly.